component/pyroscope/ebpf/metrics.go

```diff
@@ -0,0 +1,36 @@
+package ebpf
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics struct {
+	TargetsActive          prometheus.Gauge
+	ProfilingSessionsTotal prometheus.Counter
+	PprofsTotal            prometheus.Counter
+}
+
+func NewMetrics(reg prometheus.Registerer) *Metrics {
+	m := &Metrics{
+		TargetsActive: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "pyroscope_ebpf_active_targets",
+			Help: "Current number of active targets being tracked by the ebpf component",
+		}),
+		ProfilingSessionsTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "pyroscope_ebpf_profiling_sessions_total",
```

Comment:
might be interesting to have success vs fail there. This way we can alert if it's failing.